src/_assets/scss/style.scss

```diff
@@ -0,0 +1,162 @@
+$color-link: #5398e8;
+$color-text-base: #333;
+$l-header-width: 32vw;
+
+
+html {
+    box-sizing: border-box;
+    -ms-text-size-adjust: 100%;
+    -webkit-text-size-adjust: 100%;
+}
+
+*,
+*::before,
+*::after {
+    box-sizing: inherit;
+}
+
+html,
+body {
+    border: 0;
+    margin: 0;
+    padding: 0;
+    min-height: 100vh;
```

Comment:
Yeah, I'll bring in CM's stylelint (once it's done) as an NPM module, if that's cool?